src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1075,6 +1082,15 @@ def forward(
         if return_loss:
             loss = owlvit_loss(logits_per_text)
 
+        if return_base_image_embeds:
+            image_embeds = self.get_image_features(
+                pixel_values=pixel_values,
+                output_attentions=output_attentions,
+                output_hidden_states=output_hidden_states,
+                return_dict=return_dict,
+                return_projected=False,
+            )
```

Comment:
Can't this be done as follows:

```suggestion
        if return_base_image_embeds:
            image_embeds = vision_outputs[1]
```

Reply:
This way, you avoid doing a forward pass twice

Reply:
Ah, that is correct indeed. I just updated the code to eliminate the second call to self.vision_model.forward()